src/Database/LSMTree/Internal/Run.hs

```diff
@@ -202,13 +204,25 @@ fromWriteBuffer :: HasFS IO h
                 -> RunBloomFilterAlloc
                 -> RunFsPaths
                 -> WriteBuffer
+                -> WriteBufferBlobs IO h
```

Comment:
Would it be nice to have an abstraction that bundles a `WriteBuffer` with a `WriteBufferBlobs`? There seems to be some duplication in the code. Not that duplication is bad per se, but adding the abstraction gives us a single point of failure. For example, we could forget to copy a blob before/after adding it to the `RunBuilder`

Reply:
Yes I was thinking the same thing.

Reply:
I think I might leave this one for later.